package-builders/fedora-build.sh

```diff
@@ -19,4 +19,7 @@ sed -i "s/@PACKAGE_VERSION@/${VERSION}/g" /root/rpmbuild/SPECS/netdata.spec || e
 
 rpmbuild -bb --rebuild /root/rpmbuild/SPECS/netdata.spec || exit 1
 
-find /root/rpmbuild/RPMS/ -type f -name '*.rpm' -exec cp '{}' /netdata \; || exit 1
+# Copy the built packages to /netdata/artifacts (which may be bind-mounted)
+# Also ensure /netdata/artifacts exists and create it if it doesn't
+[ -d /netdata/artifacts ] || mkdir -p /netdata/artifacts
+find /root/rpmbuild/RPMS/ -type f -name '*.rpm' -exec cp '{}' /netdata/artifacts/ \; || exit 1
```

Comment:
This needs the same `chown` afterwards that is being added for Debian in #37, otherwise the same issues will arise on Linux.

Reply:
Done